cg/cli/deliver/base.py

```diff
@@ -46,8 +46,8 @@ def deliver():
 @TICKET_ID_ARG
 @DRY_RUN
 def rsync(context: CGConfig, ticket: str, dry_run: bool):
-    """The folder generated using the "cg deliver analysis" command will be
-    rsynced with this function to the customers inbox on the delivery server
+    """Will transfer a ticket folder, generated by running "cg deliver case/ticket
+    --dry-run", from the current machine to the delivery server.
```

Comment:
Hold up, why does a dry run generate a folder?

Reply:
Production purposes of projects etc. Why would be because that is the feature of those functions, and a feature that is needed. Sure, one could argue that we could have another function for this, but this is what we have in prod now, to deliver files to the inbox on hasta, to then allow us to manage the files manually before the delivery (but also a lot of other things may be done with those files, it is just very helpful to have something to collect together files)